utils/rpms/daos.spec

```diff
@@ -41,7 +41,7 @@ BuildRequires: hwloc-devel
 BuildRequires: bullseye
 %endif
 %if (0%{?rhel} >= 8)
-BuildRequires: argobots-devel >= 1.1
+BuildRequires: argobots-devel >= 1.2
```

Comment:
If you want to try to use this here (i.e. before it's landed from it's currently open PR) you will need to add:

```
PR-repos: argobots@PR-20
```

to your commit messages in this PR.  The above only works if the argobots result is "pass" i.e. green such as it is here:
![image](https://github.com/user-attachments/assets/30e14dc8-f1ca-479b-baed-fdfc2f0f2d6a)

If it's not a pass (i.e. red in Jenkins, maybe because testing failed due to some known test failure) but you still want to use it you will have to add the run number to the commit pragma such as:

```
PR-repos: argobots@PR-20:6
```

The benefit of not adding the `:<run-number>` if you can, is that you don't have to keep updating your commit message when you do additional argobots commits/runs.

Reply:
Thanks a lot for tip.
The error messages were a little bit cryptic for me.
Indeed, the packaging build was working, the unit test was failing and I was not able to find the reasons in the log files.